### ISSUE TITLE
Ensure role attribute has an appropriate value for the element (h1): A11y_MCIO-GDCO-LaborReferenceSystem_Labor Management_Technician_Organization_AI4W

### ISSUE THREAD

**Issue by Swati1700** [microsoft/playwright.dev#2085]
[A11y Agency plugin ](https://vigilant-adventure-v9qpqwn.pages.github.io/all/#plugins/a11y?q=a11y)is available to help with accessibility bug remediation.
The required tag has already been added: agency:agent=a11y:a11y-agent@company. Assign the bug to Agency to use it.
Please try it and share feedback on whether the remediation guidance was useful at CAIA11yAgency@microsoft.com.
### GitHub Tags:
#A11yTCS; #DesktopWeb; #FTP; #Win11; #A11ySev2; #A11yMAS; #WCAG4.1.2; #Name Role Value; #ES Chat; #agency:agent=a11y:a11y-agent@company; #A11yMAS; #ChromiumEdge; #AILimited; #MCIO-GDCO-LaborReferenceSystem; #E2E_MCIO-GDCO-LaborReferenceSystem_Web_June26; #A11yAuto; #AI4W;  

### Issue Subcategory 
Incorrect or missing role 

### Repro Steps 
Launch URL https://dccentral-test.microsoft.com/labor/technician/nowalsh@microsoft.com/overview/view?DDUROUTE=true --> Labor Management -->Run AI4W for fast pass

### Issue: 
Ensure role attribute has an appropriate value for the element (aria-allowed-role - https://dequeuniversity.com/rules/axe/4.10/aria-allowed-role?application=msftAI)

Target application: Labor Management - https://dccentral-test.microsoft.com/labor/technician/nowalsh@microsoft.com/overview/view?DDUROUTE=true

Element path: h1

Snippet: <'h1 _ngcontent-ng-c2425404876="" role="banner" style="margin-left: 0.5em;">Labor Management</h1'>

### How to fix: 
Fix any of the following:
  ARIA role banner is not allowed for given element

Environment: Microsoft Edge version 149.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Environment details 
Application Name: MCIO-GDCO-LaborReferenceSystem
URL: https://dccentral-test.microsoft.com/workspace?DDUROUTE=true
Chromium Edge version: Microsoft Edge version 149.0.0.0 (Official Build) (64-bit)
OS: Windows 11 Enterprise Insider Preview 26200.8655
### MAS reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)
Check out  [MAS 4.1.2: Name, Role, Value Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-4.1.2-Name-Role-Value-Guide.aspx?&xsdata=MDV8MDJ8fDU3MDA5MDBlNDdlNDQ0NTJjNTYyMDhkZWM3YjA1YmVjfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzkxNjc3NTc0MDQ2OTgwNjR8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKRFFTSTZJbFJsWVcxelgwRlVVRk5sY25acFkyVmZVMUJQVEU5R0lpd2lWaUk2SWpBdU1DNHdNREF3SWl3aVVDSTZJbGRwYmpNeUlpd2lRVTRpT2lKUGRHaGxjaUlzSWxkVUlqb3hNWDA9fDF8TDJOb1lYUnpMekU1T2pJNFptRXlZV015TFRnMU56QXROR001TmkxaU1EWTRMVFEyWVdJeU5HWmpOelkwWWw4MlltRmxOVGxpWVMxaU5HTTFMVFE0TjJZdFlqUXlZaTAxTVdZeE56YzNNekZqWldKQWRXNXhMbWRpYkM1emNHRmpaWE12YldWemMyRm5aWE12TVRjNE1URTNPRGt6T1RVMU5nPT18NjY3N2ZjNzFkYmM1NDJhNTAwOWUwOGRlYzdiMDViZTh8ZGU2OTA2ZjUzYzZkNGJmZTgxMTA4NGYwMzIxZjhhNWU%3D&sdata=emhDM3JLSG9vWDFZVmgwNjFFQmYrQ2pYc1Vnd0FZdi9iNGFtaS8zMmwvRT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-sambule%40microsoft.com) for resources and suggestions.
### Note:
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."
"Contact [CAI Accessibility Support Channel](https://teams.microsoft.com/l/channel/19:DeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%40thread.tacv2/CAI%20Accessibility%20Support%20Channel?groupId=f5eb428c-9e6e-4c16-8143-33fe3961a9ae&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for Accessibility queries on design, development, test, usability or any processes" 
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
"Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2feng.ms%2fdocs%2fcoreai%2fdevdiv%2fone-engineering-system-1es%2f1es-docs%2fes-chat%2findex&data=05%7c02%7cv-mchougule%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838215421556%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=%2B1yeHqbNhHcMuQaZNXDo%2B%2fiWxH0mZFSmWpamQk2ftws%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-mchougule%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838215450064%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=yncZXSKXaJlQ3HpXeR3TK03MNIhUe8tV2yZl49clFZY%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-mchougule%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838215467554%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=i4Fn6qPJw4eg3Ny6En0%2fPnIqw%2BpJpHuT%2f%2BZH2hE%2fRTA%3D&reserved=0)."

### Attachment:

<img width="1279" height="668" alt="Image" src="https://github.com/user-attachments/assets/a7ac471b-4117-4db5-90bd-f83fdf6f5d2b" />

https://github.com/user-attachments/assets/16760da7-d0cd-4758-a14f-2f71ade68e46
